mlrun/api/crud/tags.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2018 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import sqlalchemy.orm
+
+import mlrun.api.db.sqldb.db
+import mlrun.api.schemas
+import mlrun.api.utils.projects.remotes.follower
+import mlrun.api.utils.singletons.db
+import mlrun.api.utils.singletons.project_member
+import mlrun.config
+import mlrun.errors
+import mlrun.utils.singleton
+
+kind_to_function = {
```

Comment:
```suggestion
kind_to_function_names = {
```
Since these are not really the functions being returned.